services/core-api/app/api/mines/variances/resources/variance_list.py

```diff
@@ -116,4 +121,7 @@ def post(self, mine_guid):
             raise BadRequest('Error: Failed to create variance')
 
         variance.save()
+
+        if is_minespace_user:
```

Comment:
I think this should be `is_minespace_user()` :)

Reply:
LUUUUKE!!! 

Good catch, I'll update.